CHANGELOG_PENDING.md

```diff
@@ -1,5 +1,8 @@
 ### Improvements
 
+- [codegen/go] - Fix `ElementType` for nested collection input and output types.
```

Comment:
This seems like a bug to me. Can you explain why this is in "Improvements".

Reply:
> This seems like a bug to me. Can you explain why this is in "Improvements".

Because I put it in the wrong place :P